option optimize_for = LITE_RUNTIME;

message CsRiverLanternFloatRequest
{
 required uint32 num = 1;
}
